src/main/java/kostka/moviecatalog/MovieCatalogApplication.java

```diff
@@ -35,8 +35,8 @@
     @LoadBalanced
     public RestTemplate getRestTemplate(final RestTemplateBuilder restTemplateBuilder) {
         return restTemplateBuilder
-                .setConnectTimeout(Duration.ofSeconds(10L))
-                .setReadTimeout(Duration.ofSeconds(10L))
+                .setConnectTimeout(Duration.ofSeconds(30L))
+                .setReadTimeout(Duration.ofSeconds(30L))
```

Comment:
NIT: It would be nice to have it application.properties or something, so you can change value in production without changing code. But you probably know that already...

Reply:
Might not be worth wasting time for this educational project.